bedrock/base/tests/test_macros.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.test import TestCase, override_settings
+
+from django_jinja.backend import Jinja2
+
+jinja_env = Jinja2.get_default()
+
+
+def render(s, context=None):
+    t = jinja_env.from_string(s)
+    return t.render(context or {}).strip()
+
+
+# there is a version of this that can isolate the image tag
+#  render("{{{{ high_res_img('{0}', {1}) }}}}".format(url, optional_attributes), {"request": req})
+
+# parameterize repeated protocol macro tests to determine default eager loading and explicit lazy loading
+# @pytest.mark.parametrize("macro, expected_result", [
+#     (split, True),
+#     (billboard), True),
+#     (card, True),
+#     (hero, True),
+#     (feature card, True),
+# ])
+# def test_loading_attribute(macro, expected_result):
+#     assert is_lazy(rendered macro) contains expected_result
+
+
+@override_settings(STATIC_URL="/media/")
+class TestImage(TestCase):
+    def test_setup(self):
+        expected = '<img src="/media/test.jpg" alt="" loading="eager" />'
+        markup = render(
+            "{% from 'macros.html' import image %}{% extends 'test-template.html' %}{% block content %}{{ image(url='test.jpg') }}{% endblock %}"
```

Comment:
I don't think there's a need to `extend` any other template here. 
```suggestion
            "{% from 'macros.html' import image %}{{ image(url='test.jpg') }}"
```